File: dpo_train/translation_output/java-py/HumanEval_30_get_positive.py
from typing import List

def get_positive(l: List[int]) -> List[int]:
    return (List[int]) l.stream().filter(e -> e > 0).collect(Collectors.toList());

def check(candidate):
    assert candidate([-1, -2, 4, 5, 6]) == [4, 5, 6]
    assert candidate([5, 3, -5, 2, 3, 3, 9, 0, 123, 1, -10]) == [5, 3, 2, 3, 3, 9, 123, 1]
    assert candidate([-1, -2]) == []
    assert candidate([]) == []

def test_check():
    check(get_positive)

test_check()

File: codes/src-tgt/py-java/HumanEval_30_get_positive.java
### Python version

from typing import List

def get_positive(l: List[int]) -> List[int]:
    return [e for e in l if e > 0]

### Java version

import java.util.*;
import java.lang.reflect.*;
import org.javatuples.*;
import java.security.*;
import java.math.*;
import java.io.*;
import java.util.stream.*;
class Problem {
    public static ArrayList<Long> getPositive(ArrayList<Long> l) {
        return (ArrayList<Long>) l.stream().filter(e -> e > 0).collect(Collectors.toList());
    }
    

}

